shared/chat/conversation/input-area/normal/index.desktop.js

```diff
@@ -0,0 +1,316 @@
+    }
+    this.props.filePickerSetValue(null)
+  }
+
+  _mentionCatcherClick = () => {
+    this.props.setMentionPopupOpen(false)
+  }
+
+  _channelMentionCatcherClick = () => {
+    this.props.setChannelMentionPopupOpen(false)
+  }
+
+  render() {
+    return (
+      <Box style={{...globalStyles.flexBoxColumn, borderTop: `solid 1px ${globalColors.black_05}`}}>
+        {this.props.mentionPopupOpen && <MentionCatcher onClick={this._mentionCatcherClick} />}
+        {this.props.mentionPopupOpen && (
+          <MentionHud
+            conversationIDKey={this.props.conversationIDKey}
+            selectDownCounter={this.props.downArrowCounter}
+            selectUpCounter={this.props.upArrowCounter}
+            pickSelectedUserCounter={this.props.pickSelectedCounter}
+            onPickUser={this.props.insertMention}
+            onSelectUser={this.props.switchMention}
+            filter={this.props.mentionFilter}
+          />
+        )}
+        {this.props.channelMentionPopupOpen && <MentionCatcher onClick={this._channelMentionCatcherClick} />}
+        {this.props.channelMentionPopupOpen && (
+          <ChannelMentionHud
+            conversationIDKey={this.props.conversationIDKey}
+            selectDownCounter={this.props.downArrowCounter}
+            selectUpCounter={this.props.upArrowCounter}
+            pickSelectedChannelCounter={this.props.pickSelectedCounter}
+            onPickChannel={this.props.insertChannelMention}
+            onSelectChannel={this.props.switchChannelMention}
+            filter={this.props.channelMentionFilter}
+          />
+        )}
+        <Box style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
+          <input
+            type="file"
+            style={{display: 'none'}}
+            ref={this.props.filePickerSetRef}
+            onChange={this._pickFile}
+            multiple={true}
+          />
+          <Input
+            className={'mousetrap' /* className needed so key handler doesn't ignore hotkeys */}
+            autoFocus={false}
+            small={true}
+            style={styleInput}
+            ref={this.props.inputSetRef}
+            hintText="Write a message"
+            hideUnderline={true}
+            onChangeText={this.props.setText}
+            value={this.props.text}
+            multiline={true}
+            rowsMin={1}
+            rowsMax={5}
+            onKeyDown={this.props.onKeyDown}
+            onKeyUp={this.props.onKeyUp}
+            onEnterKeyDown={this.props.onEnterKeyDown}
+          />
+          {this.props.emojiPickerOpen && (
+            <EmojiPicker emojiPickerToggle={this.props.emojiPickerToggle} onClick={this._pickerOnClick} />
+          )}
+          {this.props.isEditing && (
+            <Text type="BodySmallPrimaryLink" onClick={this.props.onCancelEditing} style={cancelStyle}>
+              Cancel
+            </Text>
+          )}
+          <Icon onClick={this.props.emojiPickerToggle} style={styleIcon} type="iconfont-emoji" />
```

Comment:
🍳 This doesn't seem to work

Reply:
fixed in 10: https://github.com/keybase/client/pull/10755/files#diff-3697454f9700c7692f5234ab1832055dR301